saleor/plugins/avatax/plugin.py

```diff
@@ -261,6 +253,7 @@ def _calculate_checkout_shipping(
         for line in lines:
             if line["itemCode"] == "Shipping":
                 shipping_net = Decimal(line["lineAmount"])
+                discount_amount = Decimal(line.get("discountAmount", 0.0))
```

Comment:
Can you add here a comment, that `lineAmount` doesn't include an applied `discoutnAmount` but `tax` is calculated for:  net == `lineAmount` - `discountAmount`? I think this could be helpfull especially after a checking belowed lines, like:
```
shipping_net -= discount_amount
```